jgrapht-ext/pom.xml

```diff
@@ -35,6 +35,19 @@
 					</execution>
 				</executions>
 				<configuration>
+					<transformers>
```

Comment:
I don't think this change is related to the utility methods, so it should be in a separate pull request.